api/schemas/user.py

```diff
@@ -57,4 +69,40 @@ def mutate(self, info, username, password, email):
 		)
 
 		db_session.add(user)
-		db_session.commit()
+		try:
+			db_session.commit()
+		except Exception as e:
+			db_session.rollback()
+			db_session.flush()
+		raise GraphQLError("Error creating account, please try again")
+
+
+class SignInUser(graphene.Mutation):
+	class Arguments:
+		email = graphene.String(required=True, description="User's email")
+		password = graphene.String(required=True, description="Users's password")
+
+	user = graphene.Field(lambda: UserObject)
+	auth_token = graphene.String(description="Token returned to user")
+
+	@classmethod
+	def mutate(cls, _, info, email, password):
+		email = cleanse_input(email)
+		password = cleanse_input(password)
+		user = User.query.filter(User.user_email == email).first()
+
+		if user is None:
+			raise GraphQLError("User does not exist, please register")
+
+		bcrypt = Bcrypt(app)
```

Comment:
Does having separate bcrypt objects cause issues for generating/verifying passwords?  If not this PR is good to go!

Reply:
No it doesn't affect any operations